test/browser/fragments.test.js

```diff
@@ -1150,32 +1147,12 @@ describe('Fragment', () => {
 		clearLog();
 		render(<Foo condition={false} />, scratch);
 		expect(scratch.innerHTML).to.equal(html, 'rendering from true to false');
-		expectDomLogToBe(
```

Comment:
Since this test was written we converted all nested arrays to Fragments so in this test where DOM ops previously happened (due to nested arrays previously being flattened) there are now none since nested arrays become Fragments